hal/src/main/native/athena/DIO.cpp

```diff
@@ -433,18 +433,17 @@ HAL_Bool HAL_GetDIODirection(HAL_DigitalHandle dioPortHandle, int32_t* status) {
   }
 }
 
-void HAL_Pulse(HAL_DigitalHandle dioPortHandle, double pulseLengthSeconds,
+void HAL_Pulse(HAL_DigitalHandle dioPortHandle, double pulseLength,
                int32_t* status) {
   auto port = digitalChannelHandles->Get(dioPortHandle, HAL_HandleEnum::DIO);
   if (port == nullptr) {
     *status = HAL_HANDLE_ERROR;
     return;
   }
 
-  uint32_t pulseLengthMicroseconds =
-      static_cast<uint32_t>(pulseLengthSeconds * 1e6);
+  uint32_t pulseLengthUs = static_cast<uint32_t>(pulseLength * 1e6);  // μs
```

Comment:
It would conflict with the function argument name otherwise.

Reply:
This (and the other conversation) are good to be resolved. (I don't have permissions to resolve conversations since I'm not the PR author and I don't have write permissions to the repository: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations)